### ISSUE TITLE
outgoing port range

### ISSUE THREAD

**Issue by prathibhacdac** [InnovateAsterisk/Browser-Phone#542]
How to mention the outgoing port range in Browser Phone application?

**Comment by InnovateAsterisk** [comment]
I don't believe this is possible right now. Its a limitation of the `PeerConnection()` API object.
